src/invidious/routes/errors.cr

```diff
@@ -49,4 +51,4 @@ module Invidious::Routes::ErrorRoutes
     env.response.headers["Location"] = "/"
     haltf env, status_code: 302
   end
-end
+end
```

Comment:
Please also revert this change. If your text editor doesn't want to, you might need to change a config option or two.